commit 5b9b41617bf3e1282cc60f07d3d52e62399aa4ba
Merge: 22d29f1112c8 2d179e8ac02e
Author: Linus Torvalds <torvalds@linux-foundation.org>
Date:   Thu Jan 11 19:46:52 2024 -0800

    Merge tag 'docs-6.8' of git://git.lwn.net/linux
    
    Pull documentation update from Jonathan Corbet:
     "Another moderately busy cycle for documentation, including:
    
       - The minimum Sphinx requirement has been raised to 2.4.4, following
         a warning that was added in 6.2
    
       - Some reworking of the Documentation/process front page to,
         hopefully, make it more useful
    
       - Various kernel-doc tweaks to, for example, make it deal properly
         with __counted_by annotations
    
       - We have also restored a warning for documentation of nonexistent
         structure members that disappeared a while back. That had the
         delightful consequence of adding some 600 warnings to the docs
         build. A sustained effort by Randy, Vegard, and myself has
         addressed almost all of those, bringing the documentation back into
         sync with the code. The fixes are going through the appropriate
         maintainer trees
    
       - Various improvements to the HTML rendered docs, including automatic
         links to Git revisions and a nice new pulldown to make translations
         easy to access
    
       - Speaking of translations, more of those for Spanish and Chinese
    
      ... plus the usual stream of documentation updates and typo fixes"
    
    * tag 'docs-6.8' of git://git.lwn.net/linux: (57 commits)
      MAINTAINERS: use tabs for indent of CONFIDENTIAL COMPUTING THREAT MODEL
      A reworked process/index.rst
      ring-buffer/Documentation: Add documentation on buffer_percent file
      Translated the RISC-V architecture boot documentation.
      Docs: remove mentions of fdformat from util-linux
      Docs/zh_CN: Fix the meaning of DEBUG to pr_debug()
      Documentation: move driver-api/dcdbas to userspace-api/
      Documentation: move driver-api/isapnp to userspace-api/
      Documentation/core-api : fix typo in workqueue
      Documentation/trace: Fixed typos in the ftrace FLAGS section
      kernel-doc: handle a void function without producing a warning
      scripts/get_abi.pl: ignore some temp files
      docs: kernel_abi.py: fix command injection
      scripts/get_abi: fix source path leak
      CREDITS, MAINTAINERS, docs/process/howto: Update man-pages' maintainer
      docs: translations: add translations links when they exist
      kernel-doc: Align quick help and the code
      MAINTAINERS: add reviewer for Spanish translations
      docs: ignore __counted_by attribute in structure definitions
      scripts: kernel-doc: Clarify missing struct member description
      ..

diff --cc Documentation/driver-api/index.rst
index 3cc0e1a517c0,b38f81836118..eba851605388
--- a/Documentation/driver-api/index.rst
+++ b/Documentation/driver-api/index.rst
@@@ -115,8 -110,7 +110,9 @@@ available subsections can be seen below
     hte/index
     wmi
     dpll
 +   wbrf
 +   crypto/index
+    tee
  
  .. only::  subproject and html